src/background.js

```diff
@@ -469,33 +445,19 @@ const rollout = {
         }
       }
```

Comment:
This is pretty much duplicated in `runStartupHeuristics`.
I suggest moving this block of code into the `heuristics` function and renaming it `runHeuristics` or something.

Reply:
Great note. Updated!